grc/gui/canvas/param.py

```diff
@@ -156,6 +159,29 @@ def format_block_surface_markup(self):
         Returns:
             a pango markup string
         """
+
+        # TODO: is this the correct way to do this?
```

Comment:
I think we should resolve this question before merging?

Reply:
I think this actually a question for @skoslowski  since he seemed to have some idea of how to determine 'literalness'

Reply:
Well, to find out if there no variable dependencies one would have try to eval the expression in an otherwise namespace, to detect a literal `ast.literal_parse()` could be used. But I think neither is really worth it and not really appropriate here. I suggest to put in a comment and keep the check as is.